src/main/resources/templates/EditQuestion.html

```diff
@@ -66,5 +67,9 @@ <h2 th:text="'Question ID:'+${question.getId()}">Question ID</h2>
     </div>
     <p th:case="*" >Question type not implemented</p>
   </div>
```

Comment:
Maybe add a button to return home so the user is not stuck on the page

You can use the same button from SurveyFillIssue.html